mizar/daemon/interface_service.py

```diff
@@ -107,6 +108,14 @@ def ProduceInterfaces(self, request, context):
             self._QueueInterface(interface)
         return interfaces
 
+    def RemoveInterfaces(self, request, context):
+        """
+        """
+        interfaces = request
+        for interface in interfaces.interfaces:
+            self._DequeueInterface(interface)
+        return interfaces
+
```

Comment:
I don't believe we need this RPC at all.

Reply:
For delete, I thought we still need queue all the interface, and then delete one by one, just like add.

Reply:
For ADD, there is the information passed by the operator (IP address, ...etc). For DEL, do we have such a case with different interface related configurations in CNI and in Operator? I don't think so. Primarily the DEL command is self-contained, and you shall be able to delete the interface entirely in the CNI without waiting for the operator. If there are ref-counting (like the substrate) and you need to maintain that ref-count in the operator, you can make an independent call to manage that ref-count, but I really don't see that even needed.